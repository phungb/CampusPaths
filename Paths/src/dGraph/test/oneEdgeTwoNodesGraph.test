# Tests if an edge can be added to a graph with two nodes

# Create a graph
CreateGraph one_edge_graph

# Add a pair of nodes
AddNode one_edge_graph n1
AddNode one_edge_graph n2

# Add an edge
AddEdge one_edge_graph n1 n2 e1

# Print the nodes in the graph and the outgoing edges from n1
ListChildren one_edge_graph n1