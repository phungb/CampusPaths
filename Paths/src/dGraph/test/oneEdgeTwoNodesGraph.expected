# Tests if an edge can be added to a graph with two nodes

# Create a graph
created graph one_edge_graph

# Add a pair of nodes
added node n1 to one_edge_graph
added node n2 to one_edge_graph

# Add an edge
added edge e1 from n1 to n2 in one_edge_graph

# Print the nodes in the graph and the outgoing edges from n1
the children of n1 in one_edge_graph are: n2(e1)